cmd/pctl/main.go

```diff
@@ -83,7 +85,7 @@ func globalFlags() []cli.Flag {
 
 func parseArgs(c *cli.Context) (string, *client.Client, error) {
 	if c.Args().Len() < 1 {
-		return "", nil, fmt.Errorf("argument must be provided")
+		return "", nil, errors.New("<CATALOG>/<PROFILE>[/<VERSION>] must be provided")
```

Comment:
can we add a test for this?
also doesn't this affect other commands? I thought `argument must be provided` was thrown by all commands when the required argument was missing 🤔

Reply:
I can add an integration test.

Reply:
ah, and to answer your question, nope, we don't have this line anywhere in the docs. :D

Reply:
`parseArgs` is used by `get` command too 🙈

Reply:
oh wait we will never get to `if c.Args().Len() < 1 {` condition in the `get` command 👍🏻

Reply:
yes, but that part is already tested. it doesn't get into the args part because there is a different check for that in the Get command already